apps/v1/consts.go

```diff
@@ -0,0 +1,135 @@
+package v1
+
+const (
+	// DeploymentStatusReasonAnnotation represents the reason for deployment being in a given state
+	// Used for specifying the reason for cancellation or failure of a deployment
+	// This is set by deployer controller and consumed in retry and history commands.
+	DeploymentStatusReasonAnnotation = "openshift.io/deployment.status-reason"
+
+	// DeploymentPodAnnotation is an annotation on a deployment (a ReplicationController). The
+	// annotation value is the name of the deployer Pod which will act upon the ReplicationController
+	// to implement the deployment behavior.
+	// This is set by deployer controller and consumed by oc adm top command.
+	DeploymentPodAnnotation = "openshift.io/deployer-pod.name"
+
+	// DeployerPodCreatedAtAnnotation is an annotation on a deployment that
+	// records the time in RFC3339 format of when the deployer pod for this particular
+	// deployment was created.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DeployerPodCreatedAtAnnotation = "openshift.io/deployer-pod.created-at"
+
+	// DeployerPodStartedAtAnnotation is an annotation on a deployment that
+	// records the time in RFC3339 format of when the deployer pod for this particular
+	// deployment was started.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DeployerPodStartedAtAnnotation = "openshift.io/deployer-pod.started-at"
+
+	// DeployerPodCompletedAtAnnotation is an annotation on deployment that records
+	// the time in RFC3339 format of when the deployer pod finished.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DeployerPodCompletedAtAnnotation = "openshift.io/deployer-pod.completed-at"
+
+	// DesiredReplicasAnnotation represents the desired number of replicas for a
+	// new deployment.
+	// This is set by deployer controller, but not consumed by any command or internally.
+	DesiredReplicasAnnotation = "kubectl.kubernetes.io/desired-replicas"
+
+	// DeploymentAnnotation is an annotation on a deployer Pod. The annotation value is the name
+	// of the deployment (a ReplicationController) on which the deployer Pod acts.
+	// This is set by deployer controller and consumed internally and in oc adm top command.
+	DeploymentAnnotation = "openshift.io/deployment.name"
```

Comment:
This is really a replication controller name annotation inside deployer pod... We already have ownerRef there so I vote for removing this from api and replacing the origin code to use ownerRefs over this annotation.

Reply:
> This is really a replication controller name annotation inside deployer pod... We already have ownerRef there so I vote for removing this from api and replacing the origin code to use ownerRefs over this annotation.

Deprecate this one too.